### PR TITLE
Parser for /etc/corosync/corosync.conf

### PR DESCRIPTION
**Pull request by psachin**
Signed-off-by: Sachin Patil <psachin@redhat.com>

### CODE REVIEW

File: insights/parsers/corosync.py [hunk 1]
```diff
@@ -1,47 +1,171 @@
...
 
-    >>> csconfig = shared[CoroSyncConfig]
-    >>> 'COROSYNC_OPTIONS' in csconfig.data
-    True
-    >>> csconfig.options
-    ''
+CoroSyncConfig - file ``/etc/sysconfig/corosync``
+-------------------------------------------------
 
+CorosyncConf - file ``/etc/corosync/corosync.conf``
```
Comment (xiangce) [inline insights/parsers/corosync.py]: To distinguish the new Parser `CorosyncConf` with the original one, I think better`deprecate` the old one `CoroSyncConfig` and move it to the [sysconfig.py](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/sysconfig.py) and rename it as `CorosyncSysconfig`. 

For all `/etc/sysconfig/xxx` configuration files, I prefer to put them into [sysconfig.py](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/sysconfig.py)

@bfahr , @psachin - Any thoughts?

**Comment by bfahr** [comment]
@psachin this new spec will need to go through the spec approval process.

**Comment by psachin** [comment]
New specs proposal: **INSPEC-128**

File: insights/parsers/corosync.py [hunk 2]
```diff
@@ -1,47 +1,171 @@
...
+        ''
+    """
     @property
     def options(self):
         """ (str): The value of the ``COROSYNC_OPTIONS`` variable."""
         return self.data.get('COROSYNC_OPTIONS', '')
+
+
+@parser(Specs.corosync_conf)
+class CorosyncConf(Parser):
```
[the remark this answers is not in the document]
Comment (csams) [reply insights/parsers/corosync.py]: @psachin Have a look at the [configtree versions of the multipath_conf parsers](https://github.com/RedHatInsights/insights-core/blob/61a18ffda7ca0a802b52b44819d1b5c5b1e41535/insights/parsers/multipath_conf.py#L184) near the bottom of the file.

File: insights/specs/default.py [hunk 3]
```diff
@@ -122,6 +122,7 @@ def tomcat_base(broker):
     cobbler_modules_conf = first_file(["/etc/cobbler/modules.conf", "/conf/cobbler/modules.conf"])
 
     corosync = simple_file("/etc/sysconfig/corosync")
+    corosync_conf = simple_file("/etc/corosync/corosync.conf")
```
[the remark this answers is not in the document]
Comment (psachin) [reply insights/specs/default.py]: Sure

**Comment by psachin** [comment]
@bfahr @csams 

Can you please review the changes.

**Pull request merged**